Source/UserManagement/Read/DataCollectors/DataCollector.cs

```diff
@@ -10,7 +10,7 @@ namespace Read.DataCollectors
     public class DataCollector : IReadModel
     { 
         [BsonId]
-        public Guid Id { get; set; }
+        public Guid DataCollectorId { get; set; }
```

Comment:
DataCollector is an important concept in the domain, should introduce a ConceptAs<> for it in a Concepts project.

Reply:
On it